docs/compass/03-automatic-scenario-assignment.md

```diff
@@ -0,0 +1,213 @@
+```
+
+## Example
+Let assume a situation, that Integration System that is responsible for registering Runtimes, label every runtime with information about a user who triggered provisioning a Runtime. 
+Then, you can define an Assignment, that every Runtime provisioned by a given person belongs to the specific Scenario.
+
+1. Create or update `scenarios` Label Definition and specify the following scenarios: 
+* DEFAULT
+* MARKETING
+* WAREHOUSE
+
+```graphql
+mutation  {
+  createLabelDefinition(in: {key: "scenarios", schema: "{\"items\":{\"enum\":[\"DEFAULT\",\"MARKETING\",\"WAREHOUSE\"],\"maxLength\":128,\"pattern\":\"^[A-Za-z0-9]([-_A-Za-z0-9\\\\s]*[A-Za-z0-9])$\",\"type\":\"string\"},\"minItems\":1,\"type\":\"array\",\"uniqueItems\":true}"}) {
+    key
+    schema
+  }
+}
+```
+
+2. Create an Assignment, that Runtimes created by a Warehouse administrator are assigned to the `WAREHOUSE` scenario.
+```graphql
+mutation  {
+  createAutomaticScenarioAssignment(in: {scenarioName: "WAREHOUSE", selector: {key: "owner", value: "warehouse-admin@mycompany.com"}}) {
+    scenarioName
+    selector {
+      key
+      value
+    }
+  }
+}
+```
+4. Register a Runtime, with a label that indicates that it was created by the Warehouse administrator.
+```graphql
+mutation  {
+  registerRuntime(in:{name: "warehouse-runtime-1", labels:{owner:"warehouse-admin@mycompany.com"}}) {
+    name
+    labels
+  }
+}
+```
+
+As you can see from the output below, the Runtime is assigned to the `WAREHOUSE` scenario, thanks to the previously
+defined Automatic Scenario Assignment. 
+```json
+{
+  "data": {
+    "registerRuntime": {
+      "name": "warehouse-runtime-1",
+      "labels": {
+        "owner": "warehouse-admin@mycompany.com",
+        "scenarios": [
+          "WAREHOUSE"
+        ]
+      }
+    }
+  }
+}
+```
+[//]: # TODO: currently runtime is also assigned to the DEFAULT scenario, but should it?
+
+6. Remove AutomaticScenarioAssignment.
+```graphql
+mutation  {
+  deleteAutomaticScenarioAssignmentForScenario(scenarioName: "WAREHOUSE") {
+    scenarioName
+  }
+}
+```
+
+7. Fetch information about previously created Runtime, for example by listing all Runtimes
+```graphql
+query  {
+  runtimes {
+    data {
+      id
+      name
+      labels
+    }
+  }
+}
+```
+
+As you can see, Runtime was unassigned from the `WAREHOUSE` scenario.
+```json
+{
+  "data": {
+    "runtimes": {
+      "data": [
+        {
+          "id": "b5e1bf58-e8bb-4bde-a9c0-87650b0909c0",
+          "name": "warehouse-runtime-1",
+          "labels": {
+            "owner": "warehouse-admin@mycompany.com",
+            "scenarios": [
+              "DEFAULT"
+            ]
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+TODO: in the implementation, WAREHOUSE scenario is not removed
+
+The same situation occurs if you modify or remove the label `createdBy` for the Runtime.
+
+Assignment Runtime to scenarios occurs also when we define Assignment.
```

Comment:
Please add subheaders to the Example section. Here we can start another subsection